ONSdigital/census-rm-acceptance-tests-2021
# isHouseholder Flag Added To Refusal Event

# Motivation and Context
Need to know if the respondent who refused is the householder along with their name and address details
isHouseholder flag is independent of Contact and Address blocks in event JSON so even if isHouseholder is false, Contact and Address may still have details in them

# What has changed
Updated event JSON in test to include the isHouseholder flag

# How to test?
Check tests pass 
Check casev2.events to ensure the new isHouseholder flag is set

# Links
https://trello.com/c/WT0EGYyU/1150-nc-update-refusal-event-5
https://collaborate2.ons.gov.uk/confluence/display/CATD/Response+Management+Event+Dictionary?focusedCommentId=20210501#comment-20210501
